basxconnect/mailer_integration/mailchimp/mailchimp_person.py

```diff
@@ -49,3 +51,21 @@ def interests_ids(self) -> List[str]:
 
     def status(self) -> str:
         return self.raw_person["status"]
+
+    def country(self):
+        _country = pycountry.countries.get(
+            name=self.raw_person["merge_fields"].get("MMERGE11", "CH"), default=None
+        )
+        if _country:
+            return _country.alpha_2
+        else:
+            return "CH"
```

Comment:
Can we make the default country code to be taken from a setting, similar to ```PREFERRED_CURRENCIES``` in ```basxconnect.contributions.settings```? I think there should also be a list of possible choices available in django-countries.